wurst/util/StringUtils.wurst

```diff
@@ -17,6 +18,9 @@ int array width
 constant MAX_INDEX = '~' + 1
 constant DEFAULT_WIDTH = 0
 constant DEFAULT_STRING = ""
+constant int	 WC3LIMIT_FUNCTIONS_PARAMETER_COUNT  = 31 // Should be a global for whole WurstScript or StdLib at least
```

Comment:
No, this should be (and already is) inbuilt in the compiler.
You will get a warning if using more than 31 parameters.
Having this inside the vararg functions isn't a good idea imo. Especially if it's only certain ones.

Reply:
Sure, will remove it if its hardcoded there.